src/Providers/PharBuilt/PharBuilt.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * This file is part of Laravel Zero.
+ *
+ * (c) Nuno Maduro <enunomaduro@gmail.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace LaravelZero\Framework\Providers\PharBuilt;
+
+use Phar;
+
+class PharBuilt
```

Comment:
`final class Build`.

Reply:
Done. Don't know how to mock it, IDE is warning me about reflection errors?
![image](https://user-images.githubusercontent.com/26031459/53480114-6b285980-3a72-11e9-9ec3-f7b3bf3c4b9d.png)

Reply:
You pass an instance of the object, instead of the class itself. The code should NEVER be adapted for testing.

Reply:
Thanks.

So I changed to give it an object instance, and now I get this error

```
 PHPUnit\Framework\Exception  : Argument #1 (No Value) of PHPUnit\Framework\MockObject\Generator::getMock() must be a array or string
```